feedforward.cabal

```diff
@@ -1,12 +1,6 @@
--- This file has been generated from package.yaml by hpack version 0.20.0.
---
--- see: https://github.com/sol/hpack
---
--- hash: 997ed186d08ff264c26ba5714b27983ffb7c84c2a307fc559d44c6043adc7f74
-
 name:           feedforward
 version:        0.0.1
-copyright:      2018 Author name here
+copyright:      2018 Alex McLean and contributors
```

Comment:
In stack project `.cabal` file forms out of `package.yaml` file. So it is better to change line in `package.yaml` to
`copyright:           "2018 Alex McLean and contributors"`